.gitignore

```diff
@@ -4,3 +4,7 @@
 *.user
 doc
 TAGS
+
+install/
+build/
+log/
```

Comment:
I'm assuming you added these because they are being generated in your `velodyne/` folder when you call `colcon build`. You can avoid this by calling `colcon build` from the base of your workspace (above `src/`). If you have other packages that you don't want built in your workspace, you can run `colcon build --packages-select <package names>`.